PushNotifications/PushNotifications/PushNotificationsStatic.swift

```diff
@@ -0,0 +1,269 @@
+#if os(iOS)
+import UIKit
+import UserNotifications
+#elseif os(OSX)
+import Cocoa
+import NotificationCenter
+#endif
+import Foundation
+
+@objc public final class PushNotificationStatic: NSObject {
+    
+    private override init() {
+        // prevent other people initialising us
+    }
+    
+    private static var instance: PushNotifications?
+    internal static var tokenProvider = Dictionary<String, TokenProvider>()
+    
+    /**
+     Start PushNotifications service.
+
+     - Parameter instanceId: PushNotifications instance id.
+
+     - Precondition: `instanceId` should not be nil.
+     */
+    /// - Tag: start
+    @objc public static func start(instanceId: String) {
+        if (instance == nil) {
+            instance = PushNotifications(instanceId: instanceId)
+        } else if self.instance!.instanceId != instanceId {
+            print("PushNotifications.shared.start has been called before with a different instance id! before: \(self.instance!.instanceId), now: \(instanceId).")
+            print("If you want to use multiple instances, use `PushNotifications(instanceId: instanceId)` directly. For example:")
+            print("```")
```

Comment:
> Xcode console do not support Markdown.

This is intended to be read by developers such as yourself. We are trying to make a helpful message on how to use the SDK correctly, but we don't really expect you to go to production in this state. I think we had a discussion on wether this would be a `print` or a `fatalError` and it seems we opted for a `print`.